IpmiFeaturePkg/Library/IpmiBaseLibDxe/IpmiBaseLibDxe.inf

```diff
@@ -28,4 +28,4 @@
   gIpmiTransportProtocolGuid
 
 [Depex]
-  gIpmiTransportProtocolGuid
```

Comment:
Should we go ahead and do the same for MM and DXE for consistency?

Reply:
Also, should replace with TRUE instead of being empty.

Reply:
I think you are right.  For consistency we should remove it everywhere. 

I'll update the code.